service-worker.js

```diff
@@ -108,6 +108,7 @@ const precacheFiles = [
     'www/css/kiwixJS_invert.css',
     'www/css/kiwixJS_mwInvert.css',
     'www/css/transition.css',
+    'www/img/icons/kiwix-blue-32.png',
```

Comment:
Please place this entry in alphabetical order in the list. `kiwix-blue...` should come after `kiwix-60...`.